torchx/specs/api.py

```diff
@@ -334,6 +334,7 @@ class Role:
     mounts: List[Union[BindMount, VolumeMount, DeviceMount]] = field(
         default_factory=list
     )
+    nnodes_rep: str = ""  # the representation of nnodes for elastic launch
```

Comment:
can we change this to be `min_replicas: Optional[int] = None` and thus you'd use f"{min_replicas}:{num_replicas}" as the range for elastic? I think that's cleaner

Also document it

Reply:
Done

Reply:
this seems like it hasn't changed?

Reply:
It's actually removed now. It was used to let the `ray_driver.py` know the min:max numbers of nodes, but now it's not necessary.